src/phoenix/server/api/mutations/user_mutations.py

```diff
@@ -90,9 +220,18 @@ async def create_user(
         )
 
 
-def _get_user_create_error_message(error: IntegrityError) -> str:
+def _select_user_by_id(user_id: int) -> Select[Tuple[models.User]]:
+    return (
+        select(models.User).where(models.User.id == user_id).options(joinedload(models.User.role))
+    )
+
+
+def _user_crud_error_message(
+    error: IntegrityError,
+    crud: Literal["create", "modify"] = "create",
```

Comment:
it's not CRUD if it doesn't fit into Create Read Update Delete. I would call it operation or something analogous instead.

Reply:
ok